### PR TITLE
452: Get OAV allowed zoom levels from PVs

### LINKED ISSUE #452: Check OAV zoom level without hard coding acceptable values
The OAV zoom level is an enum which has values
0 = "1.0x"
1 = "2.5x"
2 = "5.0x"
3 = "7.5"
4 = "10.0x"

Currently we check a zoom level (received from GDA in OAVCentring.json) is acceptable by checking if it is in a hard coded list
``` Python    
# Level is the arbitrary level  that corresponds to a zoom percentage.
    # When a zoom is fed in from GDA this is the level it is refering to.
    level: EpicsSignal = Component(EpicsSignal, "MP:SELECT")
    allowed_zooms = ["1.0x", "2.5x", "5.0x", "7.5x", "10.0x"]
```

We should instead get the list of acceptable zoom levels from the PV directly.

### PR DESCRIPTION
**Pull request by noemifrisina**
Fixes #452

### To test:
1. Run tests against s03

### CODE REVIEW

File: src/artemis/devices/system_tests/test_oav_system.py [hunk 1]
```diff
@@ -31,3 +31,11 @@ def test_grid_overlay():
     snapshot_directory = "."
     RE = RunEngine()
     RE(take_snapshot_with_grid(oav, snapshot_filename, snapshot_directory))
+
+
+@pytest.mark.s03
+def test_get_zoom_levels():
+    my_zoom_controller = ZoomController("BL03I-EA-OAV-01:FZOOM:", name="test_zoom")
+    my_zoom_controller.wait_for_connection()
+    # raise Exception(my_zoom_controller.allowed_zoom_levels)
```
Comment (DominicOram) [inline src/artemis/devices/system_tests/test_oav_system.py:40]: Should: Can you remove commented out code please?

**Review by DominicOram — CHANGES_REQUESTED** [review]
Generally looks good, thank you. Minor comment in code. Also, can you fix the merge conflict

File: src/artemis/devices/system_tests/test_oav_system.py (hunk 1 above)
Comment (noemifrisina) [reply src/artemis/devices/system_tests/test_oav_system.py:40]: Done, forgot to delete it before committing.

**Review by DominicOram — APPROVED** [review]
Looks good, thank you

**Pull request merged**